ICRCs/ICRC-97/ICRC-97.md

```diff
@@ -0,0 +1,95 @@
+| ICRC |                   Title                    |                                      Author                                      |                Discussions                | Status |      Type       | Category |  Created   |
+|:----:|:------------------------------------------:|:--------------------------------------------------------------------------------:|:-----------------------------------------:|:------:|:---------------:|:--------:|:----------:|
+|  97  | Non-Fungible Token (NFT) Metadata Standard | Thomas (@sea-snake), Austin Fatheree (@skilesare), Dieter Sommer (@dietersommer) | https://github.com/dfinity/ICRC/issues/97 | Draft  | Standards Track |          | 2024-08-13 |
+
+# ICRC-97: Non-Fungible Token (NFT) Metadata Standard
+
+ICRC-97 is a metadata standard for the implementation of Non-Fungible Tokens (NFTs) on
+the [Internet Computer](https://internetcomputer.org).
+
+The standard is designed to allow for multiple assets with different data types with various purposes. Where additional
+asset purposes and attribute display types can be standardized in ICRC-97 extensions.
+
+## Extending the ICRC-7 Standard with Token Metadata
+
+The [`icrc7_token_metadata`](https://github.com/dfinity/ICRC/blob/main/ICRCs/ICRC-7/ICRC-7.md#icrc7_token_metadata)
+method returns the token metadata for `token_ids`, a list of token ids. Each tuple in
+the response vector comprises an optional `metadata` element with the metadata expressed as vector of `text` and `Value`
+pairs.
+
+ICRC-7 does not specify the representation of token metadata any further than that it is represented in a generic manner
+as a vector of (text, Value)-pairs. The ICRC-97 standard extends the ICRC-7 standard with a token metadata
```

Comment:
We could mention that the (text, Value)-pair rep is recursive.

Reply:
Added a sentence to mention that `Value` is recursive as defined in the ICRC-3 standard.